sites/test-site/hosts/etcd-masters.yml

```diff
@@ -0,0 +1,29 @@
+---
+name: host1
+interfaces:
```

Comment:
we will have another field under host, `power` or something else that specifies the power management details?